### ISSUE TITLE
Better direct message room management

### ISSUE THREAD

**Issue by viasux** [ajbura/cinny#237]
#### Is your feature request related to a problem? Please describe.
Room management is hell for anyone using matrix currently. I have not seen a single matrix client do room management in a "good" way imho. I run multiple chat bridges on my matrix server, meaning I have ***hundreds of chats*** which are hard to sort through. When I click the "Direct messages" section on cinny, I am presented with every single bridged chat, un-categorized.

#### Describe the solution you'd like
I would prefer if direct messages which are manually put into a space *stay in the space,* and not appear on the "direct messages" section. Also, an option to then sort direct messages by activity (https://github.com/ajbura/cinny/issues/54) would be wonderful.

#### Describe alternatives you've considered
My workaround *on element* is to put them all in spaces, and set them all to low priority, so they are hidden under my other chats in the "people" section of element. However, this is not ideal, the "low priority" option can be confusing, and imho shouldn't exist. It's a bandaid fix for a much larger problem. I see no functional workaround for this issue in cinny at this time.

**Comment by kfiven** [comment]
"Home" and "Direct messages" are part of app where you can find every chat despite them being in other spaces. For example one can pin space and create a shortcut but it still can be opened via Home. 

Also about the space, user could leave them but that doesn't close DMs with other users. 

About the sorting part, I agree that activity based sorting will be really helpful.

**Comment by ajbura** [comment]
We have DMs sorting by activity now but I don't think removing DM's from People tab (if they are added in a space) will be great as I often find myself struggling to remember that which space consist which DM and end up looking for that in People tab (and I have got same feedback from few others as well).

**Comment by viasux** [comment]
> We have DMs sorting by activity now but I don't think removing DM's from People tab (if they are added in a space) will be great as I often find myself struggling to remember that which space consist which DM and end up looking for that in People tab (and I have got same feedback from few others as well).

oh yes I wasn't advocating for that lol, don't separate dm's in a space

**Comment by viasux** [comment]
wait yes i was i dont know why i thought that would be a good idea wtf

**Comment by viasux** [comment]
OHH it was because I had bridges for telegram/discord and I wanted a way to categorize them.. I was meaning like only if they were *manually* added by the user to like a private space, for categorization

Folder functionality could solve this issue as well.

**Comment by kfiven** [comment]
You can add DMs to a space in Cinny as well but we still keep them in People tab. So if I get it correctly this issue is about "Remove DMs from People tab if they are added to a space"?

**Comment by viasux** [comment]
> You can add DMs to a space in Cinny as well but we still keep them in People tab. So if I get it correctly this issue is about "Remove DMs from People tab if they are added to a space"?

yes - also simple folder functionality could accomplish the same end goal of being able to categorize dm's

**Comment by viasux** [comment]
in fact I believe I would prefer a folder system to a system reliant on spaces..
